### ISSUE TITLE
Can this method Achieve face-mask realistic inpainting ?

### ISSUE THREAD

**Issue by francqz31** [yuanzhi-zhu/DiffPIR#7]
Hello , How are you Mr. author ! thank you for being active and answering all the questions! 

DiffPIR is so good at Face Depixelization or also what can be referred to as Restoration and it is also really good at simple inpainting tasks with animal pics. 
I was wondering if it can be trained or solely fine-tuned to do Face-mask inpainting like (https://github.com/andreas128/RePaint) and even if it can get better results than Repaint after the training or the fine-tuning ?
and if so , how can I do it ?

for reference, so it can do tasks like that and replace the mask with very accurate face
![228623149-b1ddddc1-41e5-487a-98e9-37c8c6ec6cf2](https://github.com/yuanzhi-zhu/DiffPIR/assets/117993423/459ffeea-4111-4fe9-ae02-c12c58126ecc)

**Comment by yuanzhi-zhu** [comment]
Hi @francqz31,

Both DiffPIR and RePaint are training-free methods, which means no further training or fine-tuning is required.

Specifically, in DiffPIR we can load any custom mask from an image (you have to draw the mask manually) through the load_mask option https://github.com/yuanzhi-zhu/DiffPIR/blob/main/main_ddpir_inpainting.py#L167.
(DiffPIR can be faster than RePaint without sacrificing quality, but no better results are guaranteed 🥲)

To conclude, DiffPIR can solve custom inpainting tasks by giving the image and mask pair just like in RePaint. And the generated content in the masked region depends on the prior knowledge in the pre-trained diffusion model. 

P.S: However, exercise caution with the outcome you receive👿!!
![image](https://github.com/yuanzhi-zhu/DiffPIR/assets/24802044/1742c435-00f7-4888-8300-77bc4e301f4a)

**Comment by yuanzhi-zhu** [comment]
@francqz31 Thank you for your notebook, I will add a demo in the next major update (if any).

Unfortunately, I do not know how to draw a mask efficiently. (I just brush the image with black and white
